spec/environment_helpers/enumerable_helpers_spec.rb

```diff
@@ -0,0 +1,84 @@
+RSpec.describe EnvironmentHelpers::BooleanHelpers do
```

Comment:
Wrong describe target here, not that it matters functionally.

Reply:
Yikes!